gradle.properties

```diff
@@ -15,7 +15,7 @@ org.gradle.jvmargs=-Xmx1536m
 # Android operating system, and which are packaged with your app's APK
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
-# Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+# Tell the KMM plugin where the iOS project lives
+xcodeproj=./ios
```

Comment:
BTW I tried the KMM plugin here again and it crashed every time I tried to debug. But adding this still enables a workflow to launch both platforms from one IDE when making changes to shared code.